orm_lib/inc/drogon/orm/SqlBinder.h

```diff
@@ -312,6 +312,13 @@ class DROGON_EXPORT SqlBinder : public trantor::NonCopyable
           client_(client),
           type_(type)
     {
+        if (sql[sqlLength] != '\0')
```

Comment:
I don't think this is sound; if `sql` ends at a page boundary, the access of `sql[sqlLength]` would be out-of-bounds and could cause a segfault.